### PR TITLE
Overpanel Refactor

### PR DESCRIPTION
**Pull request by aknobloch3**
- Branch name says `adding-animation`, but the main change is that addition of the overpanel code.
- Also added slight border animation for the nav items

### CODE REVIEW

**Comment by aknobloch3** [comment]
screen shot 2018-12-17 at 8 44 04 am

File: src/pages/templates/edit-overpanel.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+			<div className="title-content">
+				<div>
+					<h2>Edit Address</h2>
+					<p>Manage the address tied to your membership</p>
+				</div>
+				<button className="button button--secondary"><i className="dashing-icon dashing-icon--close"></i> Close</button>
+			</div>
+		</header>
+
+    <section className="app">
```
Comment (jrauh) [inline src/pages/templates/edit-overpanel.js]: Small thing. Aren't we killing the `app` wrapper?

File: src/sass/dashboard/main.scss [hunk 2]
```diff
@@ -71,8 +71,9 @@ $header-background: $white;
           color: $gray-100;
           font-weight: 400;
           font-size: 1.1rem;
+          @include transition(border 0.2s ease-out);
```
Comment (jrauh) [inline src/sass/dashboard/main.scss]: It does not transition for me when I open this.

File: src/sass/experimental/overpanel.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    margin-top: $space-xs; //to keep content from hiding shadow
+    max-width: $content-m;
+  }
+
+}
+
+/* Utility classes for max-width of Overpanel */
+.overpanel-s {
+  .title-content {
+    max-width: $content-s !important;
```
Comment (jrauh) [inline src/sass/experimental/overpanel.scss]: Do we have to use `!important`?

File: src/pages/templates/edit-overpanel.js (hunk 1 above)
Comment (aknobloch3) [reply src/pages/templates/edit-overpanel.js]: Oh yeah! I'll remove that. Pretty sure it isn't doing anything.

File: src/sass/experimental/overpanel.scss (hunk 3 above)
Comment (aknobloch3) [reply src/sass/experimental/overpanel.scss]: The way I structured the CSS file i needed important, but i can rearrange it to make that not needed it looks like! If I nest the `.overpanel-l`, `!important` is not needed.

**Pull request merged**